README.md

```diff
@@ -1,23 +1,28 @@
-# SlackLateX
-Bot to convert LateX code in messages into 
+# Slack LaTeX
+#### Bot to convert LaTeX code in messages into:
 
 ![screen]
 
-##How to use:
-Install required modules with:
-```
-npm install fs
-npm install request
-npm install websocket
-```
+### Setup:
+1. Clone/download this repository.
+2. Install required modules in the project folder with:
```

Comment:
This is the part I don't understand.  What project folder? Run stuff where?  I just want to be able to display LaTeX in Slack, that will show up on all devices, without my students having to install some plugin that will only work on the Desktop version.  Can't this bot be packaged as a plugin, that can be easily installed natively from within Slack?

Reply:
@TBemrose Student's won't have to install any dependencies, however, this program is supposed to be run on a server and you then will need to then create a bot on your Slack. The process is not trivial if you are not used to creating a NodeJS application server, but I am considering to work on a simpler process, but it will take some time to develop that. I will update you when I do so.

Reply:
This could be turned into a standalone app where anyone that wants to install it to their slack would only have to press one button but I wouldn't want to distribute something like that while depending on codecogs for  LateX rendering.

Reply:
Yeah, I said to myself: well, I've got the code, I'll follow these Slack help files on how to create an app ... and they were NOT helpful. They kept telling me WHAT an app could do, but never actually HOW to make one.  I've already "created the app" following the Create an App instructions, now I just need to fill it with code.  The moment they show me anywhere remotely like a text box or folder where I can start writing code, I'm off and running.  Nope.

About midnight last night I got fed up, bitched out customer support and went home.  I heard back from them this morning.  Turns out Slack doesn't have **any** apps on it at all.  They're all just shells linking to apps running elsewhere.  FAIL!